packages/OmegaPrint-Core.package/OPPrinter.class/instance/MoreExecutableCode.with.and..st

```diff
@@ -0,0 +1,11 @@
+actions
+MoreExecutableCode: aNode with: statements and: aFinalStatement
+
+	| result |
+	result := self value: statements.
+	aFinalStatement children first
+		ifNotNil: [
+			result
+				ifEmpty: [ result := self value: aFinalStatement ]
+				ifNotEmpty: [ result := result , '.' , self newline , (self value: aFinalStatement) ]].
```

Comment:
vielleicht sollte man `self value: aFinalStatement` in eine lokale Variable packen, weil es zweimal verwendet wird (DRY) und außerdem die Zeile sehr in die Laenge zieht.

Reply:
stimme dir zu!

Reply:
Finde ich auch gut

Reply:
Ich auch